pkg/driver/driver.go

```diff
@@ -767,8 +770,20 @@ func (driver *Driver) ScrubEphemeralPods(podsDirPath string) error {
 			return filepath.SkipDir /* Skip processing current directory and continue processing other directories */
 		}
 
+		log.Tracef("Walking path [%s]", fileFullPath)
+		
+		if info.IsDir() && (strings.Count(fileFullPath, string(os.PathSeparator)) - baseDepth) == 2 && info.Name() != "volumes" {
+			log.Tracef("Skipping non-volumes path [%s]", fileFullPath)
+			return filepath.SkipDir
+		}
+
+		if info.IsDir() && (strings.Count(fileFullPath, string(os.PathSeparator)) - baseDepth) == 5 {
```

Comment:
Could you also add a comment here as well?

Reply:
Done